File: src/main/java/UserPanel.java
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;

public class UserPanel {
    private WebDriver driver;
    private String mainPageUrl;
    private static final By USER_PANEL = By.id("userPanel");
    public WebElement userPanel;

    public UserPanel(WebDriver driver) {
        this.driver = driver;
        userPanel = driver.findElement(USER_PANEL);
    }

    public WebElement administrationPageLink() {
        //FIXME искать по тексту не очень
        return userPanel.findElement(By.partialLinkText("Administration"));
    }

    public Boolean isAdministrationPageLinkAbset() {
        return userPanel.findElements(By.partialLinkText("Administration")).isEmpty();
    }

    public AdministrationPage openAdministrationPage() {
        userPanel.findElement(By.partialLinkText("Administration")).click();
        return new AdministrationPage(driver);
    }
}

File: src/main/kotlin/pageModel/UserPanel.kt
package pageModel

import org.openqa.selenium.By
import org.openqa.selenium.WebDriver
import org.openqa.selenium.WebElement

class UserPanel(private val driver: WebDriver) {
    var userPanel: WebElement

    val isAdministrationPageLinkAbset: Boolean?
        get() = userPanel.findElements(By.partialLinkText("Administration")).isEmpty()

    init {
        userPanel = driver.findElement(USER_PANEL)
    }

    fun administrationPageLink(): WebElement {
        //FIXME искать по тексту не очень
        return userPanel.findElement(By.partialLinkText("Administration"))
    }

    fun openAdministrationPage(): AdministrationPage {
        userPanel.findElement(By.partialLinkText("Administration")).click()
        return AdministrationPage(driver)
    }

    companion object {
        private val USER_PANEL = By.id("userPanel")
    }
}
